test/opengl/conv_select.cpp

```diff
@@ -4,6 +4,8 @@
 #include "Halide.h"
 #include <stdio.h>
 
+#include "./testing.h"
```

Comment:
A habit of mine -- intends to hint to a reader that the .h file is in the local directory rather than elsewhere in the include path.  But not important.  I switched them to "testing.h"